presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java

```diff
@@ -109,18 +109,15 @@ private static HiveDataSource getTextFileDataSource()
         return createResourceDataSource(format(tableNameFormat, "textfile"), String.valueOf(ThreadLocalRandom.current().nextLong(Long.MAX_VALUE)), "com/facebook/presto/tests/hive/data/all_types/data.textfile");
     }
 
-    private static HiveDataSource getParquetDataSource()
-    {
-        return createResourceDataSource(format(tableNameFormat, "parquet"), String.valueOf(ThreadLocalRandom.current().nextLong(Long.MAX_VALUE)), "com/facebook/presto/tests/hive/data/all_types/data.parquet");
-    }
-
-    private static HiveDataSource getOrcDataSource()
+    public static void populateDataToHiveTable(String tableName)
```

Comment:
This is a bit hacky.
Probably more elegant solution would be to have a DataSource that internally creates hive table. Populates it. 
And then returns location of data files from that table.
But then you would need to pass text file and table schma to such DS.
It would be too verbose.

I think your solution is good enough.